### PR TITLE
Adding noise covariances to KF and initial pose

### PR DESCRIPTION
**Pull request by calculusrobotics**
next step is to make it calculate stuff but that's scary so we're stopping here :)

### CODE REVIEW

File: InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java [hunk 1]
```diff
@@ -631,7 +631,10 @@ public double getRightVelocity_mps() {
     }
 
 	public Trajectory getFirstPickupTrajectory() {
-        return pickupTrajectoryChooser.getSelected().getFirstPickupTrajectory();
+        Trajectory traj = pickupTrajectoryChooser.getSelected().getFirstPickupTrajectory();
```
Comment (cpostbitbuckets) [inline InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java:634]: It’s generally bad form for a getter function to have side effect.

Comment (calculusrobotics) [reply InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java:634]: bUt It WoRkS

Comment (cpostbitbuckets) [reply InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java:634]: How about doing it in autonomous init?

```java
        // reset nav system, turn on LL leds
        navigationSubsystem.reset();
        navigationSubsystem.setInitialPose(driveSubsystem.getFirstPickupTrajectory().getInitialPose());
```

Comment (cpostbitbuckets) [reply InfiniteRecharge/src/main/java/frc/robot/subsystem/drive/DriveSubsystem.java:634]: or maybe make a separate driveSubsystem method for getInitailPose() that just returns `driveSubsystem.getFirstPickupTrajectory().getInitialPose()`

**Pull request merged**